src/auto/application.rs

```diff
@@ -368,6 +398,12 @@ impl<O: IsA<Application>> GtkApplicationExt for O {
 
     #[cfg(any(feature = "v3_24", feature = "dox"))]
     fn connect_property_screensaver_active_notify<F: Fn(&Self) + 'static>(&self, f: F) -> SignalHandlerId {
+#[cfg(any(feature = "v3_24", feature = "dox"))]
+        unsafe extern "C" fn notify_screensaver_active_trampoline<P, F: Fn(&P) + 'static>(this: *mut gtk_sys::GtkApplication, _param_spec: glib_sys::gpointer, f: glib_sys::gpointer)
+        where P: IsA<Application> {
```

Comment:
Wrong ident

Reply:
Actually we don't need the condition anymore. I'll remove it its generation. :)